app/views/hiring_staff/organisations/schools/edit.html.haml

```diff
@@ -12,11 +12,12 @@
       = f.govuk_error_summary
 
       %h2.govuk-heading-l
-        = t('.title')
+        = t('.title', organisation: organisation_type_basic(@organisation))
 
       = f.govuk_text_area :description,
-        label: { text: t('helpers.fieldset.organisation_form.description'), size: 's' },
-        hint_text: t('helpers.hint.organisation_form.description'),
+        label: { text: t('helpers.fieldset.organisation_form.description', organisation: organisation_type_basic(@organisation)), size: 's' },
+        hint_text: t('helpers.hint.organisation_form.description', organisation: organisation_type_basic(@organisation)),
```

Comment:
Feature/system tests are time-expensive, so I'm OK with the logic of 'trust' vs 'school' going untested.